sem1/hw8/8_1/8_1.cpp

```diff
@@ -0,0 +1,96 @@
+#include "list.h"
+
```

Comment:
8_1.cpp:30:2: error: ‘cout’ was not declared in this scope
  cout << "Enter the amount of students: ";
  ^
8_1.cpp:31:2: error: ‘cin’ was not declared in this scope
  cin >> totalAmount;
  ^
8_1.cpp:52:11: error: ‘endl’ was not declared in this scope
   cout << endl << "Enter the number of the current student: ";
           ^
8_1.cpp:73:12: error: ‘endl’ was not declared in this scope
    cout << endl << "The number of student: " << i + 1 << endl;
            ^
8_1.cpp:81:11: error: ‘endl’ was not declared in this scope
   cout << endl << "These students should be expelled: ";